scripts/build-storybooks.js

```diff
@@ -148,7 +148,9 @@ const handleExamples = async deployables => {
     const out = p(['built-storybooks', d]);
     const cwd = p(['examples', d]);
 
-    await exec(`yarn`, [`build-storybook`, `--output-dir=${out}`, '--quiet'], { cwd });
+    await exec(`yarn`, [`build-storybook`, `--output-dir=${out}`, '--quiet', '--debug-webpack'], {
```

Comment:
is the `--debug-webpack` necessary? i'd think it would generate a lot of noise

Reply:
yeah we can revert that, used it for debugging